src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelListItem.tsx

```diff
@@ -95,10 +95,7 @@ export const TrackPanelListItem = (props: TrackPanelListItemProps) => {
   } = props;
 
   useEffect(() => {
-    const { defaultTrackStatus } = props;
-    if (trackStatus !== defaultTrackStatus) {
-      updateGenomeBrowser(trackStatus);
-    }
+    updateGenomeBrowser(trackStatus);
```

Comment:
I completely forgot this code, so am rather confused here. Are we saying that every list item in the track panel sends individual message to update genome browser when it mounts? Isn't it something that the TrackPanel component has already done?

Reply:
That's true! In which case this is completely redundant. It seems to work okay without this block.